packages/h3/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@bull-board/h3",
+  "version": "5.11.0",
+  "description": "A H3 server adapter for Bull-Board dashboard.",
+  "keywords": [
+    "bull",
+    "bullmq",
+    "redis",
+    "h3",
+    "adapter",
+    "queue",
+    "monitoring",
+    "dashboard"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/felixmosh/bull-board.git",
+    "directory": "packages/h3"
+  },
+  "license": "MIT",
+  "author": "genu",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "clean": "rm -rf dist"
+  },
+  "dependencies": {
+    "@bull-board/api": "5.11.0",
+    "@bull-board/ui": "5.11.0",
+    "ejs": "^3.1.9"
+  },
+  "peerDependencies": {
+    "h3": "^1.10.0"
```

Comment:
why `peerDeps`? this adapter is using `h3`, so it should be in `deps`. (otherwise the build is failing)